library/src/main/java/com/cube/storm/ui/fragment/StormBottomTabsFragment.java

```diff
@@ -83,7 +83,6 @@ protected void loadPages(@NonNull TabbedPageCollection collection)
 			}
 		}
 
-		setTabItemContentDescriptions();
```

Comment:
Is that for the title or maybe the ADA? I think we should not remove it.

Reply:
That method is triggered on onTabSelected method, so I removed setTabItemContentDescriptions() call from loadPages because it is not necessary. That method sets the tabs content descriptions.

Reply:
Yes, but if i remember correctly, the description of any view is used for ADA stuff, so i think we should keep it.

Reply:
@juancoob  if you open the app, without changing tabs, is the onTabSelected method called?

Reply:
You were right guys, onTabSelected is not called when the app start, so I'm going to leave it 😃